web-app/js/portal/ui/openlayers/layer/NcWms.js

```diff
@@ -278,7 +279,7 @@ OpenLayers.Layer.NcWms = OpenLayers.Class(OpenLayers.Layer.WMS, {
     },
 
     _getTimeParameter: function(dateTime) {
-        return dateTime.clone().utc().format('YYYY-MM-DDTHH:mm:ss.SSS');
+        return dateTime.clone().utc().format('YYYY-MM-DDTHH:mm:ss.SSS') + 'Z';
```

Comment:
You'll want to escape the Z character. I think it's square brackts:
`console.log(time.utc().format('YYYY-MM-DDTHH:mm:ss.SSS[Z]'));`